README.md

```diff
@@ -373,6 +379,12 @@ val peers = client.getPeerList(true);
 SemVer version = client.getVersion();
 ```
 
+#### getAccountList
+```java
+val accounts = client
```

Comment:
Please provide the type - not everyone uses `Lombok`.

Reply:
done